packages/sui-jest/bin/sui-jest-browser.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+const program = require('commander')
+const runner = require('../src/runner.js')
+
+// Setup default env vars
+process.env.BABEL_ENV = 'test'
+process.env.NODE_ENV = 'test'
+
+program
+  .option('-W, --watch', 'Run in watch mode')
+  .option('--ci', 'Run a Firefox headless for CI testing')
```

Comment:
There isn't a plural version for Firefox. This should be one of the following:

1 - Run a Firefox headless browser for CI testing
2 - Run Firefox in headless mode for CI testing

Reply:
Thanks, I should check/update this description in order this is wrong I guess. Thanks again for detect it!